github/version.go

```diff
@@ -0,0 +1,3 @@
+package github
+
+const Version = "4.26.1"
```

Comment:
I don't love the idea of introducing this file. I'd rather use a solution in which we use linker flags to set the version corresponding to the git tag in our build system. 

I'm imagining using the `ldflags` option in Goreleaser [here](https://goreleaser.com/customization/build/#builds) in order to auto-increment the build version. Does that make sense?